easybuild/easyconfigs/d/double-conversion/double-conversion-3.1.4-GCCcore-8.3.0.eb

```
@@ -14,15 +14,24 @@ checksums = ['95004b65e43fefc6100f337a25da27bb99b9ef8d4071a36a33b5e83eb1f82021']
 
 builddependencies = [
     ('binutils', '2.32'),
-    ('SCons', '3.1.1'),
+    ('CMake', '3.15.3'),
 ]
 
-installopts = "DESTDIR=%(installdir)s prefix='' && "
-installopts += "mkdir %(installdir)s/include && cp double-conversion/*.h %(installdir)s/include"
+separate_build_dir = True
+
+local_buildtype = '-DCMAKE_BUILD_TYPE=Release '
+configopts = [
+    local_buildtype + c for c in (
+        '',
+        '-DCMAKE_POSITION_INDEPENDENT_CODE=ON -DCMAKE_STATIC_LIBRARY_SUFFIX_CXX=_pic.a',
+        '-DBUILD_SHARED_LIBS=ON'
+    )
+]
 
 sanity_check_paths = {
-    'files': ['include/double-conversion.h', 'include/utils.h', 'lib/libdouble-conversion.a',
-              'lib/libdouble-conversion.%s' % SHLIB_EXT, 'lib/libdouble-conversion_pic.a'],
+    'files': ['include/double-conversion/double-conversion.h', 'include/double-conversion/utils.h',
+              'lib/libdouble-conversion.a', 'lib/libdouble-conversion_pic.a',
+              'lib/libdouble-conversion.%s' % SHLIB_EXT],
```

I would expand the sanity checks
```suggestion
    'files': ['include/double-conversion/%s.h' % h for h in ['bignum', 'cached-powers', 'diy-fp', 'double-conversion',
                                                             'fast-dtoa', 'fixed-dtoa', 'ieee', 'strtod', 'utils']] +
             ['lib/libdouble-conversion.%s' % e for e in ['a', SHLIB_EXT]] + ['lib/libdouble-conversion_pic.a'],
```

---

I actually disagree here. It should be a sanity check that stuff is installed to correct location. Checking "include/double-conversion/double-conversion.h" is enough for the headers (one can assume the build systems installs the rest of the headers to the same location)

As for the list comprehension in lib/*: With the same argument as you have in https://github.com/easybuilders/easybuild-easyconfigs/pull/9659#discussion_r393257522 I'd keep it a simple list for clarity

---

Ok, I will not insist as it is a minor issue. But just to be clear, the point is to detect a partial installation of the files that may occur due to network or i/o issues. It might be very unlikely, but it does not cost much to add those checks.